tests/hacker_rank/test_list_comprehensions.py

```diff
@@ -0,0 +1,19 @@
+from code_challenges.hacker_rank.list_comprehensions import create_cuboids
+
+
+def test_create_cuboids_n2():                    
+    actual = create_cuboids(1, 1, 1, 2)                                                   
+
+    assert actual == [[0, 0, 0], [0, 0, 1], [0, 1, 0], [1, 0, 0], [1, 1, 1]], f"Returned: {actual}, instead of [[0, 0, 0], [0, 0, 1], [0, 1, 0], [1, 0, 0], [1, 1, 1]]"
```

Comment:
So it would look like
```
actual = create_cuboids(1, 1, 1, 2)                                                   
expected = [[0, 0, 0], [0, 0, 1], [0, 1, 0], [1, 0, 0], [1, 1, 1]]


assert actual ==  expected, f"Returned: {actual}, instead of {expected}"
```